public/static/locales/fr/form-builder.json

```diff
@@ -329,5 +329,12 @@
   "translateTitle": "Saisir la traduction",
   "unlockPublishing": "Soumettre",
   "unlockPublishingDescription": "Pour déverrouiller la publication, fournissez les coordonnées de la personne de qui vous relevez. ",
-  "untitledForm": "[FR] Unnamed form file"
+  "untitledForm": "[FR] Untitled form",
```

Comment:
This should be reverted to "[FR] Unnamed form file"

Reply:
Thanks will fix that merge conflict